src/scrub.c

```diff
@@ -343,8 +347,8 @@ static int scrub_object(char *filename, const struct opt_struct *opt,
                 }
             }
             break;
-        case FILE_LINK:
```

Comment:
Can we really omit this case? Shouldn't it stay and be handled the same as the next one?